modules/core/02-client/types/params_legacy.go

```diff
@@ -0,0 +1,31 @@
+package types
```

Comment:
```suggestion
/*
NOTE: Usage of x/params to manage parameters is deprecated in favor of x/gov
controlled execution of MsgUpdateParams messages. These types remains solely
for migration purposes and will be removed in a future release.
[#3621](https://github.com/cosmos/ibc-go/issues/3621)
*/

package types
```

Add tracking issue for removal of these types.